saw-remote-api/python/saw_client/__init__.py

```diff
@@ -648,6 +648,26 @@ def prove(goal: cryptoltypes.CryptolJSON,
         pr.counterexample = None
     return pr
 
+def eval_int(expr: cryptoltypes.CryptolJSON) -> int:
+    """Atempts to evaluate the given expression as a concrete integer.
+    """
+    conn = __get_designated_connection()
+    res = conn.eval_int(expr).result()
+    v = res['value']
+    if not isinstance(v, int):
+        raise ValueError(str(v) + " is not an integer")
+    return v
+
+def eval_bool(expr: cryptoltypes.CryptolJSON) -> int:
```

Comment:
```suggestion
def eval_bool(expr: cryptoltypes.CryptolJSON) -> bool:
```

Reply:
It's curious that the Mypy run in CI didn't catch this. Testing locally: if I remove the `isinstance` check, I get the expected type error ("returning Any from a function declared to return bool"), but I don't get any error if the `isinstance` check mismatches the declared return type.

Reply:
A `bool` is an `int`:

```python
>>> isinstance(False, int)
True
>>> isinstance(True, int)
True
>>> issubclass(bool, int)
True
```

Reply:
Ah, that would explain it. TIL. Thanks!